### PR TITLE
FR-1963 Amend document display in Contested

### PR DESCRIPTION
**Pull request by ravinderpal**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/FR-1963

### Change description ###
- create 10 new collection for both applicant and respondent 
- added hearingDetails textArea
- created statement/Affidavit document type 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/contested/json/FixedLists/FixedLists.json [hunk 1]
```diff
@@ -2278,40 +2278,47 @@
     "ListElement": "Expert Evidence",
     "DisplayOrder": 16
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "FR_s_caseDocumentType",
+    "ListElementCode": "Statement/Affidavit",
+    "ListElement": "Statement/Affidavit",
```
Comment (shaunganley) [inline definitions/contested/json/FixedLists/FixedLists.json:2285]: my understanding is that nothing to do with the fixed list is changing?

File: definitions/contested/json/CaseTypeTab/CaseTypeTab-respSol-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    "TabFieldDisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/02/2021",
+    "CaseTypeID": "FinancialRemedyContested",
+    "TabID": "Applicant Documents",
+    "TabLabel": "Applicant Documents",
+    "TabDisplayOrder": 18,
+    "CaseFieldID": "appFormEExhibitsCollection",
+    "TabFieldDisplayOrder": 2
```
Comment (shaunganley) [inline definitions/contested/json/CaseTypeTab/CaseTypeTab-respSol-nonprod.json:18]: will these appear above the old collections in the applicant documents tab? if so, do you not need to update the display order for the old collections?

File: definitions/contested/json/FixedLists/FixedLists.json (hunk 1 above)
Comment (ravinderpal) [reply definitions/contested/json/FixedLists/FixedLists.json:2285]: I noticed a discrepancy in the list provided. Chased it up and turns out they want an addition type.

File: definitions/contested/json/CaseTypeTab/CaseTypeTab-respSol-nonprod.json (hunk 2 above)
Comment (ravinderpal) [reply definitions/contested/json/CaseTypeTab/CaseTypeTab-respSol-nonprod.json:18]: Good spot. Updated old collections order!

**Review by jakub-kozlowski — APPROVED** [review]

**Review by shaunganley — APPROVED** [review]

**Comment by ravinderpal** [comment]
Temporary closing PR to release resources

**Pull request merged**